DailyCodingProblem/Program.cs

```diff
@@ -211,5 +213,32 @@ static int CheckDecimalValueRange(int value)
                 return value;
             }
         }
+
+        /// <summary>
+        /// Given the mapping a = 1, b = 2, ... z = 26, and an encoded message, count the number of ways it can be decoded.
+        /// For example, the message '111' would give 3, since it could be decoded as 'aaa', 'ka', and 'ak'.
+        /// You can assume that the messages are decodable. For example, '001' is not allowed.
+        /// </summary>
+        static int Problem7(string message)
+        {
+            int path = 0;
+
+            if (message == string.Empty)
```

Comment:
To cover null strings, it would be cool to use string.NullOrEmpty
```suggestion
            if (string.IsNullOrEmpty(message))
```